app/controllers/admin_users_account_suspensions_controller.rb

```diff
@@ -0,0 +1,32 @@
+# -*- encoding : utf-8 -*-
+# app/controllers/admin_users_account_suspensions_controller.rb:
+# Controller for suspending user accounts
+#
+# Copyright (c) 2017 UK Citizens Online Democracy. All rights reserved.
+# Email: hello@mysociety.org; WWW: http://www.mysociety.org/
+
+class AdminUsersAccountSuspensionsController < AdminController
+  before_filter :set_banned_user
+  before_filter :set_suspension_reason
+
+  def create
+    if @banned_user.update(ban_text: @suspension_reason)
```

Comment:
Should also clear profile photo

Reply:
Yeah, this is happening currently. If you want to get this merge without this for now it is fine